### PR TITLE
Show due date

### LINKED ISSUE #95: Show due date in daily task listing

### PR DESCRIPTION
**Pull request by jkhabra**
Resolves #95

### CODE REVIEW

File: ui/src/components/Tasks/DailyTasks/index.tsx [hunk 1]
```diff
@@ -95,6 +98,11 @@ const DailyTaskRow: React.FC<DailyTask & { isFetching: boolean }> = ({ isFetchin
     setModalOpen(false);
   };
 
+  const dateFromNow = () => {
+    const d = dayjs(p.date).fromNow();
+    return d.split(' ')[1] === 'hours' ? 'a day ago' : d;
+  };
```
Comment (bitspook) [inline ui/src/components/Tasks/DailyTasks/index.tsx]:
```suggestion
  const dateFromNow = () => {
    const day = dayjs(p.date);
    
    if (day.isToday()) {
      return 'Today';
    }
    
    return day.fromNow();
  };
```

File: ui/src/entities/Task.ts [hunk 2]
```diff
@@ -44,8 +47,7 @@ export const createDailyTasks = async (payload: DailyTask[]): Promise<undefined>
 };
 
 export const fetchManyDailyTasks = async (): Promise<FetchTasksResponse> => {
-  const date = new Date();
-  const today = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`;
+  const today = dayjs(new Date()).format('YYYY-MM-DD');
```
Comment (bitspook) [inline ui/src/entities/Task.ts:50]: :+1:

**Review by bitspook — CHANGES_REQUESTED** [review]

File: ui/src/components/Tasks/DailyTasks/index.tsx [hunk 3]
```diff
@@ -95,6 +98,15 @@ const DailyTaskRow: React.FC<DailyTask & { isFetching: boolean }> = ({ isFetchin
     setModalOpen(false);
   };
 
+  const dateFromNow = () => {
+    const day = dayjs(p.date);
+    if (!day.isBefore(dayjs(), 'day')) {
```
Comment (bitspook) [inline ui/src/components/Tasks/DailyTasks/index.tsx:103]: Is this working as expected? Shouldn't this be `if (day.isAfter(dayjs(), 'day')`?

Comment (jkhabra) [reply ui/src/components/Tasks/DailyTasks/index.tsx:103]: it is working with `isBefore`

**Review by bitspook — APPROVED** [review]

**Pull request merged**